grpclog/grpclog.go

```diff
@@ -218,3 +226,22 @@ func (l *GrpcLog) Marshal() ([]byte, error) {
 	}
 	return bytes, nil
 }
+
+type hasURN interface {
+	GetUrn() string
```

Comment:
We don't do this consistently, but acronyms should be capitalized in Go:
```suggestion
	GetURN() string
```

Reply:
No can do, this is designed to match stuff like [this](https://github.com/pulumi/pulumi/blob/3b9f5c09b7ac2f8cc111d1f8bf77b184ba71d6d5/sdk/proto/go/provider.pb.go#L797-L802).